README.md

```diff
@@ -33,11 +55,11 @@ Its options are described below.
 ```
   --verbose                       Show logs.
 
-  --start TEXT                    The date to begin analyzing the project at - has to be an iso date.
```

Comment:
I didn't want the `start` argument for the mappings to get confused with the `start_time` attribute of the work items, so I renamed the arguments for the mappings like so: `start` --> `after` and `end` --> `before`. Plus, the before/after naming convention is more consistent with evergreen.py: https://github.com/evergreen-ci/evergreen.py/blob/27864f17401c58e6311bd1e366a7679db4505172/src/evergreen/api.py#L358